### ISSUE TITLE
Handle repositories with more than one binary

### ISSUE THREAD

**Issue by biluohc** [nabijaczleweli/cargo-update#75]
```
Updating clippy from https://github.com/rust-lang-nursery/rust-clippy
    Updating git repository `https://github.com/rust-lang-nursery/rust-clippy`
error: multiple packages with binaries found: clippy, clippy_workspace_tests    
```

**Comment by nabijaczleweli** [comment]
I can hardly see how this is even remotely related to `cargo-update`?

**Comment by mati865** [comment]
@nabijaczleweli if you want to update clippy installed from git you will see this error.

**Comment by nabijaczleweli** [comment]
That's all cool and good, the same thing happens if you try to install it via raw `cargo`, so I'm still failing to see how it's `cargo-update`'s fault/problem?

Here's the output from my machine (where I don't have `clippy`):
```
$ cargo install --git https://github.com/rust-lang-nursery/rust-clippy
    Updating git repository `https://github.com/rust-lang-nursery/rust-clippy`
error: multiple packages with binaries found: clippy, clippy_workspace_tests
```

**Comment by biluohc** [comment]
You can install clippy from git by running the follow Command:
```sh
 cargo install --git https://github.com/rust-lang-nursery/rust-clippy clippy  
```
Are you willing to deal with this situation?

**Comment by nabijaczleweli** [comment]
Wow we're getting somewhere, that's a fixable issue

**Comment by nabijaczleweli** [comment]
Acquiesce a check on `master`, if y'all would. Will release if successful 👍.

**Comment by nabijaczleweli** [comment]
As a ping on this: if you have anything against the release, speak now, or forever hold your silence (i.e. will release ca. Monday if nobody protests).

**Comment by mati865** [comment]
Works fine, thanks.

**Comment by nabijaczleweli** [comment]
Released in [`v1.5.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v1.5.0).
